library/alloc/src/vec.rs

```diff
@@ -500,6 +502,7 @@ impl<T> Vec<T> {
     /// vec.reserve(10);
     /// assert!(vec.capacity() >= 11);
     /// ```
+    #[doc(alias = "realloc")]
```

Comment:
That makes a lot of items for `realloc`. It kinda kills the purpose, no?

Reply:
You think? Rust's bounds checking removes disallows a general-purpose realloc in favor of a few specialized functions. I would assume someone looking for "I need to realloc this slice" would be interested in finding the various specialized methods Rust has to perform reallocation.

Reply:
Not sure. Well, I'll let it up to you.